conf/configuration.go

```diff
@@ -47,6 +51,8 @@ type Configuration struct {
 
 // Load will construct the config from the file `config.json`
 func Load(configFile string) (*Configuration, error) {
+	viper.SetConfigType("json")
```

Comment:
This is what happens without this code when there are no configuration files, which is perfectly valid if you only use environment variables:

```